code/modules/projectiles/boxes_magazines/internal/_cylinder.dm

```diff
@@ -53,6 +53,7 @@
 	ammo_type = /obj/item/ammo_casing/caseless/mime
 	caliber = "mime"
 	max_ammo = 6
+	exists = FALSE
```

Comment:
make it true on lethals

Reply:
also it's not on ammo box it's on `/obj/item/ammo_casing/caseless/mime`

Reply:
~~make this false on all types of energy casings as well~~

Reply:
technically energy guns have no recoil and the mass they shoot is minimal, so that would make sense.

Reply:
It's also a joke and should absolutely be a separate PR

Reply:
> make it true on lethals

Lethal mime rounds aren't real bullets that exist.